src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKAnnotationPluginDescriptor.java

```diff
@@ -400,6 +400,7 @@ public void validateAndResolvePlugins() throws CommandLineException {
                     .peek(a -> {
                         if (!founderIds.isEmpty()) a.setFounderIds(founderIds);
                         if (pedigreeFile != null) a.setPedigreeFile(pedigreeFile);
+                        a.validateArguments();
```

Comment:
I see how mixing individually provided founderIDs *and* a pedigree file might make sense for annotations that only require founderIDs, but is there a case where an annotation such as PossibleDeNovo, that requires trios,  does something useful with founderIDs that are provided in addition to a pedigree file ? If not, it seems like should be rejected.

Reply:
What if you requested both the PossibleDenovo and ExcessHet annotations, you then provide both a pedigree file and FounderIDs. What is the proper behavior here? PossibleDenovo has adequate input and so does ExcessHet. Should that case be considered invalid, should it warn? Should it be impossible to ever provide a FounderIDs argument if PossibleDenovo has been requested by the tool?